src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java

```diff
@@ -546,7 +546,7 @@ final Charset charset() {
     protected String outputTimestamp;
 
     /**
-     * The algorithm to use for selecting which files to compile.
+     * The algorithm to use for selecting which files to compile. The default values are implemented in `incrementalCompilationConfiguration()`
```

Comment:
The reader of this documentation cannot access easily the `incrementalCompilationConfiguration()` source code. I think that it is better to not add this reference.